tests/utils/test_conn.py

```diff
@@ -0,0 +1,14 @@
+from deployme.utils.conn import (
+    find_free_port,
+    is_port_in_use,
+)
+
+
+def test_find_free_port():
+    free_port = find_free_port()
+    assert isinstance(free_port, int)
+
+
+def test_port_in_use():
+    free_port = find_free_port()
+    assert not is_port_in_use(free_port)
```

Comment:
It seems we should also check the case when the port is already in use too, don't we?